src/backend/function/Fusion.Resources.Functions/Deployment/function.template.json

```diff
@@ -164,6 +164,10 @@
             {
               "name": "provision_position_queue",
               "value": "[parameters('settings').queues.provisionPosition]"
+            },
+            {
+              "name": "total_batch_time_in_minutes",
```

Comment:
Check config in azure that this is updated. We have a pr slot for this function that you can check